.github/workflows/codespell.yml

```
@@ -3,9 +3,9 @@ name: Codespell
 
 on:
   push:
-    branches: [master]
+    branches: [main]
```

or should it be 
```suggestion
    branches: [main, gh-pages]
```
? and below too